docs/metadata.md

```diff
@@ -77,3 +77,24 @@ Cozy-Client will:
 
 ⚠️ These values will override any values that your document may already have.
 
+### Specific case of io.cozy.files
+
+`io.cozy.files` attribut cozyMetadata is protected by cozy-stack and can't be saved directly.
+
+This is the case for :
+* `cozyMetadata.sourceAccount`
+* `cozyMetadata.sourceAccountIdentifier`
+
+Since cozy-client 34.11.0, you can use a direct attribut in the document to save it.
+
+```
+const client = new client()
+client.save(
+  _type: 'io.cozy.files',
+  sourceAcount: '12ab3c',
+  sourceAccountIdentifier: 'john@example.org',
+  ...
+)
```

Comment:
Plutôt quelque chose du genre je pense ?
```suggestion
client.save({
  _type: 'io.cozy.files',
  sourceAcount: '12ab3c',
  sourceAccountIdentifier: 'john@example.org'
})
```

Reply:
By seeing this, I realize this is troublesome to handle for an update:
```
const doc = await client.query(Q('io.cozy.files').getById('myid'))
const newDoc = updateDocWithStuff(doc)
await client.save(newDoc) 
```

If the `cozyMetadata.sourceAccount` was changed in the process,  the dev must think to pass it directly as a `newDoc.sourceAccount` to update it, and not the received `newDoc.cozyMetadata.sourceAccount`. 
Maybe passing `cozyMetadata.sourceAccount` in the [FileCollection](https://github.com/cozy/cozy-client/blob/master/packages/cozy-stack-client/src/FileCollection.js#L543) would make more sense.
What do you think @Crash-- ?

Reply:
You can not update the sourceAccount / sourceAccountIdentifier. Cozy-Stack doesn't allow it. This data is too much sensitive.

Reply:
Forgotten brackets added.

For the other point, that's the main reason of this problem, you can't pass cozyMetadata.sourceAccount.
So the second best option to pass sourceAccount directly.